client-side/C#/StreetSmartArcGISPro/StreetSmartArcGISPro/VectorLayers/VectorLayer.cs

```diff
@@ -142,7 +142,8 @@ public void GeoJsonToNew()
     {
       if (_geoJsonOld != null)
       {
-        GeoJson = _geoJsonOld;
+          // HBR: is not needed anymore
+//        GeoJson = _geoJsonOld;
       }
```

Comment:
Is there still a need for this function (GeoJsonToNew()) then? It seems to be doing nothing beside the removed line.